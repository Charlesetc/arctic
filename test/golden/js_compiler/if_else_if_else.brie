define main [
  if false [
    1
  ] else if false [
    2
  ] else if false [
    3
  ] else [
    4
  ]
]
