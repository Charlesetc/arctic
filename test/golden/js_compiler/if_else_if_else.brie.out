const global_functions = {1: {}};
function main() {
	if (false) {return 1;} else {if (false) {return 2;} else {if (false) {return 3;} else {return 4;};};};
}
console.log(main());
var test/golden/js_compiler/if_else_if_else.brie_main = new_closure(1)